lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -55,15 +55,16 @@ class StartUrl extends Gatherer {
       return {isReadFailure: true, reason: manifest.value.start_url.debugString};
     }
 
+    // @ts-ignore - TODO(bckenny): should actually be testing value above, not debugString
```

Comment:
this is a bug we let slip in after the last rewrite. Having a `debugString` doesn't mean there's a failure necessarily. It could have been a warning but then fell back to the documentURL.

This means we're failing sites that use one of the fallbacks even though it's technically allowed. Above should be checking if there's not a `manifest.value.start_url.value` for the failure case, leaving here to pass on the `value` to try to load it (and probably passing along any `debugString` as a warning they messed up the relative URL or whatever).